packages/fast-components-react-base/src/image/README.MD

```diff
@@ -0,0 +1,31 @@
+# Implementation
+
+## Implementation example
+```javascript
+
+<!-- Returns <img />-->
+<Image 
+    src='https://placehold.it/539x300/2F2F2F/171717'
+    alt='Placeholder with grey background and dimension watermark without any imagery'
+/>
+
+<!-- Returns <img /> using SrcSet-->
+<Image 
+    src='https://placehold.it/539x300/2F2F2F/171717'
+    srcSet='https://placehold.it/1399x600/2F2F2F/171717 1399w, https://placehold.it/1083x500/2F2F2F/171717 1083w, https://placehold.it/767x400/2F2F2F/171717 767w'
+    sizes='100vw'

```

Comment:
I don't think sizes gets used

Reply:
HTMLImageElement.sizes also is a property

Reply:
Also experimental, it might work in a few browsers but not across the board

Reply:
It’s a little awkward to write up, but we might want to recommend the [default `sizes` approach we landed on for WordPress](https://make.wordpress.org/core/2015/11/10/responsive-images-in-wordpress-4-4/):

```html
sizes="(max-width: {{image-width}}px) 100vw, {{image-width}}px"
``` 

That effectively says “this image will occupy 100% of the viewport until it reaches its natural size, at which point it will occupy its natural size.” So:

```html
sizes="(max-width: 1399px) 100vw,  1399px"
```

`sizes` isn’t super built for “defaults,” where it should always be describing the exact space the `img` will occupy across breakpoints, but this at least removes the potential for upscaling.

Reply:
> I think it's fine that we surface this functionality for people, but I don't think this is a good case in setting default.

Oh, _totally_ agreed, then. Setting a `sizes` default of any kind is basically going against the spec—and browsers are gonna treat an omitted value as `100vw` anyway, so the component doesn’t really benefit from being opinionated there.